pkg/asset/manifests/azure/cluster.go

```diff
@@ -200,6 +200,10 @@ func GenerateClusterAssets(installConfig *installconfig.InstallConfig, clusterID
 			TenantID: session.Credentials.TenantID,
 		},
 	}
+	if installConfig.Config.CredentialsMode == types.ManualCredentialsMode {
```

Comment:
The azure session has an AuthenticationType field: https://github.com/openshift/installer/blob/master/pkg/asset/installconfig/azure/session.go#L51
Maybe that's a better test here? Can't the credentials mode be Manual and the installer still authenticate with client/secret?
```suggestion
	if session.AuthType == azic.ManagedIdentityAuth {
```

Reply:
I think the credentials mode will also be Manual when using certificate creds.

Reply:
Fixed.